virtualfish/environment.fish

```diff
@@ -0,0 +1,56 @@
+function __vfext_environment_activate --on-event virtualenv_did_activate
+    if test -f $VIRTUAL_ENV/virtualfish-environment
+        for line in (cat $VIRTUAL_ENV/virtualfish-environment)
+            # Skip empty lines and comments
+            if not string length -q $line
+                or test (string sub -s 1 -l 1 $line) = "#"
+                continue
+            end
+
+            set key (echo $line | cut -d = -f 1)
+            set value (echo $line | cut -d = -f 2-)
+
+            # Preserve existing env var with shared name
+            if set -q $key
+                set -xg __VF_ENVIRONMENT_OLD_VALUE_$key $$key
```

Comment:
You use `set -gx [...]` in some places and `set -xg [...]` in others. Perhaps standardize on `set -gx [...]` for consistency? (I told you my comments would be minor!) 😉